wasm/test_page/js/index.js

```diff
@@ -38,22 +38,48 @@ const _prepareTranslateOptions = (paragraphs) => {
   return translateOptions;
 };
 
+const textToHTML = (text) => {
+  const div = document.createElement('div');
+  div.appendChild(document.createTextNode(text));
+  return div.innerHTML;
+};
+
 const translateCall = () => {
-  const text = document.querySelector("#input").value + "  ";
+  const text = document.querySelector("#input").value;
   if (!text.trim().length) return;
 
-  const paragraphs = text.split("\n");
+  const paragraphs = [textToHTML(text)]; // escape HTML 
```

Comment:
@jelmervdl Motivation of splitting the content with a new line separator was to create multiple entries to translate for the batch translation api. This change now will create only one entry in the batch. Right? Basically, `textToHTML` should be called for each new line separated entry of `text` object.

Reply:
That makes sense! I've restored it (and updated the code to insert each entry in its own div in the output so that the sentence ids are at least unique inside the div. The 'mousover' highlight code now makes use of this.)

Reply:
Thanks for restoring.